portal/models/auth.py

```diff
@@ -152,7 +153,9 @@ def notify(self, data):
 
         # Use celery asynchronous task 'post_request'
         kwargs = {'url': self.callback_url, 'data': formdata}
-        res = post_request.apply_async(kwargs=kwargs)
+        celery = create_celery(current_app)
```

Comment:
I do think we want to use a singleton pattern, rather than create another celery on every request such as this.